ams/server/src/main/java/com/netease/arctic/server/dashboard/DashboardServer.java

```diff
@@ -214,6 +214,9 @@ public EndpointGroup endpoints() {
             get(
                 "/tables/catalogs/{catalog}/dbs/{db}/tables/{table}/branches",
                 tableController::getTableBranches);
+            post(
```

Comment:
I think `cancel` is okay, and the reason why using`tables/catalogs/{catalog}/dbs/{db}/tables/{table}/optimizing-processes/{processId}/cancel` instead of `/tables/catalogs/{catalog}/dbs/{db}/tables/{table}/cancel-process/{processId}` is to be a RESTful API.

Reply:
OK, i will use "/tables/catalogs/{catalog}/dbs/{db}/tables/{table}/cancel-process/{processId}" instead.

Reply:
I have changed the api, plz review it again.